### ISSUE TITLE
<html> element does not have a [lang] attribute

### ISSUE THREAD

**Issue by gedenata** [facebook/docusaurus#2471]
## 🐛 Bug Report

If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly.

Have you read the Contributing Guidelines on issues?
yes.

### To Reproduce

* When deploy the index.html to server there is no `lang` attribute, or
* View Page Source the website `<!DOCTYPE html><html lang=""><head><meta charSet="utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=edge" />`

### Expected behavior

* https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/accessibility/html-has-lang.js
* https://dequeuniversity.com/rules/axe/3.3/html-has-lang

### Your Environment

* Docusaurus version used: "docusaurus": "^1.14.4"
* Version 80.0.3987.149 (Official Build) (64-bit)
* macOS Mojave V.10.14.6
